components/public-api/gitpod/v1/organization.proto

```diff
@@ -86,20 +86,62 @@ message OnboardingSettings {
 }
 
 message OrganizationSettings {
-  bool workspace_sharing_disabled = 1;
-  string default_workspace_image = 2;
+  optional bool workspace_sharing_disabled = 1;
```

Comment:
Note how fields IDs stay stable, we just add the `update_` fields at the bottom for usage on the `UpdateSettingsRequest`.

Reply:
Reverted